include/nbl/video/IPhysicalDevice.h

```diff
@@ -267,6 +267,20 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
             uint8_t storageBufferViewAtomic : 1u; // imageBuffer
             uint8_t accelerationStructureVertex : 1u;
 
+            SFormatBufferUsage()
+                : isInitialized(0)
+            {}
+
+            SFormatBufferUsage(core::bitflag<asset::IBuffer::E_USAGE_FLAGS> usages)
+                : isInitialized(1),
+                vertexAttribute(usages.hasFlags(asset::IBuffer::EUF_VERTEX_BUFFER_BIT)),
+                bufferView(usages.hasFlags(asset::IBuffer::EUF_UNIFORM_TEXEL_BUFFER_BIT)),
+                storageBufferView(usages.hasFlags(asset::IBuffer::EUF_STORAGE_TEXEL_BUFFER_BIT)),
+                // Other way of getting these flags?
+                accelerationStructureVertex(usages.hasFlags(asset::IBuffer::EUF_ACCELERATION_STRUCTURE_BUILD_INPUT_READ_ONLY_BIT)),
```

Comment:
I think that's correct @Erfan-Ahmadi, anything wrong?

Reply:
It is fine,
although I will change all these structs later